common/script/script-api/src/main/java/org/thingsboard/script/api/tbel/TbUtils.java

```diff
@@ -1154,7 +1154,7 @@ public static void raiseError(String message) {
     }
 
     public static void raiseError(String message, Object value) {
-        String msg = value == null ? message : message + " for value " + value;
+        String msg = value == null ? message : message + " A value of " + value + " is invalid.";
```

Comment:
Now `raiseError("Some error", X)` will result into "Some error A value of X is invalid.". Rollback this change.